data/persistence/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
     id("kotlin-multiplatform")
     id("com.android.library")
-    id("com.squareup.sqldelight")
+//    id("com.squareup.sqldelight")
```

Comment:
What's the issue here?

Reply:
plugin is added but no database definition found. (since it is unused yet)